shared/src/commonMain/kotlin/com/myprojectname/app/ui/components/PetCardBig.kt

```diff
@@ -33,20 +33,18 @@ import com.myprojectname.app.ui.theme.Typography
 import io.kamel.image.KamelImage
 import io.kamel.image.asyncPainterResource
 
+@OptIn(ExperimentalMaterial3Api::class)
```

Comment:
The opt in is in file level (look at the top of the file) no need to add it to the method. If you have noticed that the IDE is complaining is because there is a problem with the IDE linter but not in the code. Hope Jetbrains solve it any time soon.